qemu/patches/4.0.x/0003-memory-backend-file-nvdimm-support-read-only-files-a.patch

```diff
@@ -0,0 +1,43 @@
+From bf71d7d8e04e85517578c021804c77739f9fed72 Mon Sep 17 00:00:00 2001
```

Comment:
OOI, I presume this patch is heading or has already landed upstream? Nice though :-)

Reply:
we are working on that, see https://patchew.org/QEMU/20190708211936.8037-1-julio.montes@intel.com/